packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -144,6 +144,7 @@ class _CupertinoSwitchState extends State<CupertinoSwitch> with TickerProviderSt
         onChanged: widget.onChanged,
         vsync: this,
         dragStartBehavior: widget.dragStartBehavior,
+        enableOnOffLabels: MediaQuery.onOffSwitchLabelsEnabled(context)
```

Comment:
I switched _(pun semi-intended)_ the position of the word "enabled" so as to avoid suggesting that this method somehow enables on/off labels. That said, I'm worried that having a slightly different name may be confusing. What do you think?

I also remove the word "switch" from the equivalent parameter to RenderCupertinoSwitch since it seems redundant. Not sure if it's better to use the exact same name ("enableOnOffSwitchLabels") everywhere or to adapt for intuitiveness, as I've tried to do.